screens/order_complete.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import { StyleSheet, Text, View, Dimensions, Image} from 'react-native';
+import { Font } from 'expo';
+
+export default class Completed extends React.Component {
+    state = {
+      fontLoaded: false,
+    };
+    async componentDidMount() {
+      await Font.loadAsync({
+        'Montserrat-SemiBold': require('./assets/fonts/Montserrat-SemiBold.otf'),
+        'Montserrat-Regular': require('./assets/fonts/Montserrat-Regular.otf'),
+      });
+      this.setState({ fontLoaded: true });
+    }
+
+    render() {
+      return (
+        <View>
+        {
+          this.state.fontLoaded ? (
+            <View style={styles.container}>
+              <Image source ={require('./images/scart.png')} style={styles.cart} />
+              <Image source ={require('./images/personicon.png')} style={styles.person} />
+              <View style={styles.peachBox}/>
+              <Text style={styles.header}>Your order has been placed!</Text>
+              <View style={styles.purpleBox}/>
+              <Text style={styles.subHeader}>Next Steps:</Text>
+              <Text style={{left: 25, zIndex: 100000, position: 'absolute', width: 330, fontFamily: 'Montserrat-Regular', fontSize: 22, color: '#FFFFFF', top: 280}}>Look for Notifications about your delivery on the Home Page</Text>
```

Comment:
Move styles into the `styles` section - you shouldn't have inline styles, makes the code harder to read